src/sentry/static/sentry/app/views/releases/list/releaseCard.tsx

```diff
@@ -39,13 +39,17 @@ const ReleaseCard = ({
   selection,
   showHealthPlaceholders,
 }: Props) => {
-  const {version, commitCount, lastDeploy, authors, dateCreated} = release;
+  const {version, commitCount, lastDeploy, authors, dateCreated, versionInfo} = release;
 
   return (
     <StyledPanel reloading={reloading ? 1 : 0}>
       <PanelBody>
         <StyledPanelItem>
           <HeaderLayout>
+            <PackageNameColumn>
+              <ColumnTitle>{t('Package Name')}</ColumnTitle>
+            </PackageNameColumn>
```

Comment:
Should this column only show for releases that have packages? Many web projects won't have package names in their release data.

Reply:
We discussed this internally and decided that it is always better to display this column, even if the release does not provide a package name. In my opinion, it can be confusing for the users to see this info only sometimes